### PR TITLE
LPD-41420 Product Configuration List Rel entity creation

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-41420
This PR only consist the creation of  the ProductConfigurationListRel entity.

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:upgrade

**Comment by dsitu** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:upgrade

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
@andrea-ale-sbarra we can send this first if it looks good.

**Comment by dsitu** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListRelServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+	@SystemEvent(type = SystemEventConstants.TYPE_DELETE)
+	public CPConfigurationListRel deleteCPConfigurationListRel(
+			CPConfigurationListRel cpConfigurationListRel)
+		throws PortalException {
+
+		_checkCommerceCatalog(
+			cpConfigurationListRel.getCPConfigurationListId(),
+			ActionKeys.UPDATE);
+
+		return cpConfigurationListRelLocalService.addCPConfigurationListRel(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListRelServiceImpl.java:62]: this should call the delete method

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListRelServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+	public CPConfigurationListRel deleteCPConfigurationListRel(
+			long cpConfigurationListRelId)
+		throws PortalException {
+
+		CPConfigurationListRel cpConfigurationListRel =
+			cpConfigurationListRelLocalService.fetchCPConfigurationListRel(
+				cpConfigurationListRelId);
+
+		if (cpConfigurationListRel == null) {
+			throw new PrincipalException();
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListRelServiceImpl.java:76]: shouldn't this be a NoSuch..Exception?

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Ehi @dsitu I've catched a couple of typos.
Can you also please some tests? 
Thank you!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**